docs/02. Controllers.md

```diff
@@ -0,0 +1,205 @@
+# Controllers
+
+ZfrRest comes with an abstract controller that allows you to automatically dispatch based on verbs.
+
+## Simple example
+
+This example allows you to handle the "get" and "delete" method. If a request with another verb is sent,
+ZfrRest will automatically trigger a `405 Method Not Allowed` exception, with the `Allow` header fill with
+acceptable verbs.
+
+```php
+use ZfrRest\Mvc\Controller\AbstractRestfulController;
+
+class UserController extends AbstractRestfulController
+{
+    public function get(array $params)
+    {
+        // ...
+    }
+
+    public function delete(array $params)
+    {
+        // ...
+    }
+}
+```
+
+All your controllers must extend `ZfrRest\Mvc\Controller\AbstractRestfulController`, each method must be the
```

Comment:
Well, you're still able to extend other controllers if you don't want ZfrRest to play there